test/putTransferSpec.js

```diff
@@ -1018,103 +1016,7 @@ describe('PUT /transfers/:id', function () {
         .end()
     })
 
-  /* Subscriptions */
-  it('should trigger subscriptions', function * () {
```

Comment:
Why is that test under `describe('GET /accounts/:id/transfers (websocket)', function () {`?